ts/src/bingx.ts

```diff
@@ -2039,9 +2041,18 @@ export default class bingx extends Exchange {
         const amount = this.safeString2 (order, 'origQty', 'q');
         const filled = this.safeString2 (order, 'executedQty', 'z');
         const statusId = this.safeString2 (order, 'status', 'X');
+        let feeCurrencyCode = this.safeString2 (order, 'feeAsset', 'N');
+        const feeCost = this.safeStringN (order, [ 'fee', 'commission', 'n' ]);
+        if (feeCurrencyCode === undefined) {
+            if (side === 'buy') {
```

Comment:
@ndubel are you sure this logic applies to perps as well? I placed some shorts and longs and the fee was always paid in USDT 
![image](https://github.com/ccxt/ccxt/assets/43336371/8bb8fae6-aa18-4bd7-9065-828b937196e8)

Reply:
sorry, it probably my mistake

Reply:
@ndubel I think we can apply that logic for spot though
![image](https://github.com/ccxt/ccxt/assets/43336371/16602609-d6b8-4325-b72c-2c66982ce817)